Makefile

```diff
@@ -36,6 +36,37 @@ ct-tls:
 
 edoc:
 	@rebar3 edoc skip_deps=true
+	@$(MAKE) patch-doc
```

Comment:
I don't see why another Make process is necessary here. Why not just run the patch code here, after rebar3 edoc? Alternatively define targets as

    edoc: rebar3-edoc patch-doc

    rebar3-edoc:
        @rebar3 edoc skip_deps=true

    patch-doc:
        ....

Reply:
Since running edoc is really fast, I'd prefer just combining the targets in to one `edoc` including the patching.

Reply:
The order is important here, and the targets in this variant might run parallell (I found out since ran into that problem..)

Reply:
One more idea: Use make dependencies `doc/%.md: src/%.erl`. Then, running edoc becomes a no-op if no files are changed.

Reply:
> The order is important here

Good point. OK, leave it as it is then (if you don't want to combine all doc into one target).

Reply:
I can pull them together to same target